msm-sitemap.php

```diff
@@ -627,10 +627,16 @@ public static function build_root_sitemap_xml() {
 	}
 
 	public static function get_sitemap_post_id( $year, $month, $day ) {
+		$ymd = self::get_date_stamp( $year, $month, $day );
+
 		$sitemap_args = array(
-			'year' => $year,
-			'monthnum' => $month,
-			'day' => $day,
+			'date_query' => array(
+				array(
+					'before' => sprintf( '%s 00:00:00', $ymd ),
+					'after' => sprintf( '%s 00:00:00', $ymd ),
```

Comment:
The before and after here are the same - should `before` not be `&s 23:59:59`, or does the use of `inclusive` apply to the entire day rather than just the second ([the docs aren't clear](https://codex.wordpress.org/Class_Reference/WP_Query#Date_Parameters))?